morphir/runtime/src/org/finos/morphir/runtime/TestSummary.scala

```diff
@@ -25,7 +29,8 @@ case class TestSummary(
   override def toString =
     s"$report\n" +
       s"$overallCounts\n" +
-      s"Overal Status - $result"
+      s"Overall Status - $result\n" +
```

Comment:
Not sure how this got through before but this should be `\r\n` or better yet `System.lineSeparator`.

Reply:
swapped to `System.lineSeparator()`